### ISSUE TITLE
standard_init_linux.go:190: exec user process caused "exec format error"

### ISSUE THREAD

**Issue by i3anaan** [monicahq/monica#1290]
**Describe the bug**
The monicahq container crashes immediately after launch. It was never able to get past this.

**Console output**
```
sudo docker-compose up
Creating network "monica_default" with the default driver
Creating monica_monicahq_1 ... done
Attaching to monica_monicahq_1
monicahq_1  | standard_init_linux.go:190: exec user process caused "exec format error"
monica_monicahq_1 exited with code 1
```
**docker-compose.yml file** (I tried to create a minimal reproducable case)
```
version: "2"
services:
  monicahq:
    build: .
    image: monicahq/monicahq
#    links:
#      - mysql
    ports:
      - 8080:8080
    env_file: .env
    volumes:
      - ./persist/storage/app/public:/var/www/monica/storage/app/public

#  mysql:
#    image: mysql:5.7
#    environment:
#      - MYSQL_ROOT_PASSWORD=sekret_root_password
#      - MYSQL_DATABASE=monica
#      - MYSQL_USER=homestead
#      - MYSQL_PASSWORD=secret
#    volumes:
#      - ./persist/mysql:/var/lib/mysql
```

**Which version are you using:**
 - Commit: `commit 845d65b8217ae0732042fc271918587bfcadc373` (master)
 - Running on a raspberry pi:
   - ` Linux raspberrypi 4.9.59-v7+ #1047 SMP Sun Oct 29 12:19:23 GMT 2017 armv7l GNU/Linux`

**Additional context**
- Since a mysql container is not available for the raspberry pi, I initially configured it to use a postgres db. However upon launching it I got this issue and gradually reverted back to the standard configuration, which is why the mysql container is disabled.
- Setting `APP_DEBUG=true` had no additional effect.
- I have other docker containers running on the same system
  - `Docker version 18.03.0-ce, build 0520e24`
  - `docker-compose version 1.20.1, build 5d8c71b`

**Comment by i3anaan** [comment]
Okay so this was actually part my side:
The error was thrown because the container downloaded using `pull` was not an arm container, while the hardware was arm architecture.

What is really confusing to me, and maybe this is something to look into, is that I got an error message saying that no mysql container was found for my architecture. I accepted this and switched to a postgres database, but because of this I also assumed that `docker-compose pull` checked if the architecture matches, and otherwise throw that error.

So maybe you got something configured wrong regarding architectures on dockerhub, else this was simply my user error.
